src/stacks-hierarchy/Root_PurchaseConfirmationScreen/components/SelectPaymentMethodSheet.tsx

```diff
@@ -418,6 +403,30 @@ const SinglePaymentMethod: React.FC<PaymentMethodProps> = ({
   );
 };
 
+function defineSelection(
```

Comment:
`defineSelection` seems to be too generic, I would pick something more specific, for example `getPaymentSelection`